numpy/core/tests/test_deprecations.py

```diff
@@ -125,7 +126,7 @@ def assert_not_deprecated(self, function, args=(), kwargs={}):
                         exceptions=tuple(), args=args, kwargs=kwargs)
         """
         self.assert_deprecated(function, num=0, ignore_others=True,
-                        exceptions=tuple(), args=args, kwargs=kwargs)
+                               exceptions=tuple(), args=args, kwargs=kwargs)
```

Comment:
Would be better not to commit these random style changes in the same PR as a feature.

If you're not aware, try using `git add -p` which lets you choose which changes you actually want to commit

Reply:
Thanks for the tip! Should I reverse these (it is significant).

Reply:
Most of these come from PEP8, very few are PyCharm specific.

Reply:
Random as in unrelated to the change, not random stylistically. Keeping diffs small makes blames easier in future (ie, "which commit introduced this deprecation" is harder to answer if you touch a lot of lines). So yes, reversing these would be good.

Reply:
Done!